src/components/Card.js

```diff
@@ -28,11 +29,13 @@ export default function Creepypasta({
             <Title>{title}</Title>
             <Wordcount>{wordcount} Words</Wordcount>
           </CardContent>
-          <ReadMoreButton>
-            <ReadMoreButtonText to="/fullcreepypasta" state={{ id: id }}>
-              Read
-            </ReadMoreButtonText>
-          </ReadMoreButton>
+          <ReadMorePosition>
```

Comment:
I don't think it is necessary to wrap a div around the button to put it in the right position
The button should get in the same position with `position: absolute` 🤔